src/components/biggive-campaign-card-filter-grid/biggive-campaign-card-filter-grid.tsx

```diff
@@ -268,6 +276,17 @@ export class BiggiveCampaignCardFilterGrid {
     });
   };
 
+  /**
+   * We've seen desktop Safari jump to this input when it's focused at times when that's
+   * unhelpful, so on a few occasions we proactively blur it.
+   */
+  private unfocusTextInput() {
+    const input = this.el.shadowRoot?.querySelector('.input-text') as HTMLInputElement | undefined;
+    if (input) {
+      input.blur();
+    }
```

Comment:
not worth changing now this is merged but could be simplified to `input?.blur();`